suite-native/device-manager/src/components/ConnectButton.tsx

```diff
@@ -0,0 +1,125 @@
+import { useMemo } from 'react';
+import { Platform } from 'react-native';
+import { FadeInUp, FadeOutUp, LinearTransition } from 'react-native-reanimated';
+import { useSelector } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { TrezorDevice } from '@suite-common/suite-types';
+import {
+    selectHasRunningDiscovery,
+    selectInstacelessUnselectedDevices,
+    selectIsDeviceConnected,
+    selectSelectedDevice,
+} from '@suite-common/wallet-core';
+import { EventType, analytics } from '@suite-native/analytics';
+import { AnimatedBox, Button } from '@suite-native/atoms';
+import { FeatureFlag, useFeatureFlag } from '@suite-native/feature-flags';
+import { IconName } from '@suite-native/icons';
+import { Translation, TxKeyPath } from '@suite-native/intl';
+import {
+    AuthorizeDeviceStackRoutes,
+    RootStackParamList,
+    RootStackRoutes,
+    StackToStackCompositeNavigationProps,
+} from '@suite-native/navigation';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { ANIMATION_DURATION } from './DeviceList';
+import { useDeviceManager } from '../hooks/useDeviceManager';
+
+type ConnectButtonProps = {
+    onSelectDevice: (device: TrezorDevice) => void;
+};
+
+const buttonWrapperStyle = prepareNativeStyle(utils => ({
+    paddingTop: utils.spacings.sp32,
+    paddingHorizontal: utils.spacings.sp16,
+}));
+
+type NavigationProp = StackToStackCompositeNavigationProps<
+    RootStackParamList,
+    RootStackRoutes.AppTabs,
+    RootStackParamList
+>;
+
+export const ConnectButton = ({ onSelectDevice }: ConnectButtonProps) => {
+    const { applyStyle } = useNativeStyles();
+    const hasDiscovery = useSelector(selectHasRunningDiscovery);
+    const isDeviceConnected = useSelector(selectIsDeviceConnected);
```

Comment:
nitpick: `selectIsNoPhysicalDeviceConnected` would be slightly better here if you have multiple trezors. Otherwise, we show the "Connect a different Trezor" button if you have the Trezor connected, but select a different view-only trezor.

Reply:
https://github.com/trezor/trezor-suite/pull/20561/commits/c353251404bb2720fbd55afd0e372a141972c489 like this?

Reply:
yes 👍